src/components/LooMap/LocateMapControl.js

```diff
@@ -4,7 +4,8 @@ import { MapControl, withLeaflet } from 'react-leaflet';
 
 import styles from '../css/loo-map.module.css';
 
-class LocateMapControl extends MapControl {
+/** @component */
+export class LocateMapControl extends MapControl {
```

Comment:
I'm not sure if we should expose leaflet components to styleguidist as I think they will only render when they're a child of the map component.

Reply:
Yeah I don't think so either. Other than `Box` (which was mainly introduced to demonstrate `styled-system` and `emotion` working), the style guide is basically blank. I suspect, for example, we also won't want to see `EntryForm`, `MarkerClusterGroup`, and `ProtectedRoute`,

Reply:
aha all good, i suppose we can blacklist components we don't want to appear somehow?
not important right now

Reply:
Yeah we totally can :slightly_smiling_face: I already added bunch to the `ignore` list in `styleguide.config.js`